app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -198,6 +209,11 @@ private void loadSubscriptions() {
                     emptyView.updateVisibility();
                     progressBar.setVisibility(View.GONE);
                 }, error -> Log.e(TAG, Log.getStackTraceString(error)));
+
+        if (UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE)
+            filterMsg.setVisibility(View.VISIBLE);
+        else
+            filterMsg.setVisibility(View.GONE);
```

Comment:
Please add braces. To see more checkstyle violations, please have a look at the output of the `static-analysis` CI task.